scripts/extract_planning.R

```diff
@@ -2,7 +2,7 @@
 
 
 # 1. change this to the correct project name in our GIS folder and delete this comment
-dir_project <- 'sern_peace_fwcp_2023'
+dir_project <- 'sern_fraser_2023'
```

Comment:
you should be burning the gpkg directly to the GIS folder so this is the name of the QGIS directory!!  We are getting funky mixing QGIS and R projects but there are many benefits I will show you.  We have brand new functions to help us do this in a reproducable way too!!

Reply:
mmm that's odd I did change it to `sern_lchl_nerc_fran_2023` in this commit here https://github.com/NewGraphEnvironment/fish_passage_fraser_2023_reporting/pull/12/commits/328ff4024e63824e62b03909d7738186c70f4c9c but I guess it didn't change??

Reply:
![Screen Shot 2024-02-07 at 9 35 32 AM](https://github.com/NewGraphEnvironment/fish_passage_fraser_2023_reporting/assets/151881303/258cb63b-be30-428e-b101-3b324395fe5c)